<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUsers">
    <property name="title">
     <string>Users</string>
    </property>
    <addaction name="actionCreate_Users"/>
    <addaction name="actionView_Users"/>
   </widget>
   <addaction name="menuUsers"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreate_Users">
   <property name="text">
    <string>Create Users</string>
   </property>
  </action>
  <action name="actionView_Users">
   <property name="text">
    <string>View Users</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
